lib/engine/step/issue_shares.rb

```diff
@@ -36,7 +36,7 @@ def issuable_shares(entity)
         # Done via Sell Shares
         num_shares = entity.num_player_shares - entity.num_market_shares
         bundles = entity.bundles_for_corporation(entity)
-        share_price = @game.stock_market.find_share_price(entity, :left).price
```

Comment:
could just wrap the function in Array( so you don't need [] here

Reply:
7c7439ac